### DISCUSSION TITLE
flowgate add adapter

### DISCUSSION THREAD

**Discussion by yigitguvenir** [community/community#38513] [Other Feature Feedback, Questions, & Ideas]
Could not resolve dependencies for project com.vmware.flowgate:adapter-sample:jar:1.0.0: The following artifacts could not be resolved: com.vmware.flowgate:common-restclient:jar:1.2.0, com.vmware.flowgate:flowgate-common:jar:1.2.0: Failure to find com.vmware.flowgate:common-restclient:jar:1.2.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

PLS FİX

**Comment by david-engelmann** [comment]
You will get better responses posting this question in [the flowgate repo's](https://github.com/vmware/flowgate) issues section. This discussion section is for general github questions.
